documentation/source/index.rst

```diff
@@ -1,37 +1,216 @@
+
+cocotb requires a simulator to simulate the HDL design
+and has been used with a variety of simulators on Linux, Windows and Mac OS.
+Please check the :ref:`simulator-support` page for specifics.
+
+A (possibly older) version of cocotb can be used live in a web-browser using `EDA Playground <https://www.edaplayground.com>`_.
+
+
+************************
+How is cocotb different?
+************************
+
+cocotb encourages the same philosophy of design re-use and randomized testing as UVM, however is implemented in Python rather than SystemVerilog.
+
+With cocotb, VHDL/Verilog/SystemVerilog are normally only used for the design itself, not the testbench.
+
+cocotb has built-in support for integrating with the `Jenkins <https://jenkins.io/>`_ continuous integration system.
+
+cocotb was specifically designed to lower the overhead of creating a test.
+
+cocotb automatically discovers tests so that no additional step is required to add a test to a regression.
+
+All verification is done using Python which has various advantages over using SystemVerilog or VHDL for verification:
+
+* Writing Python is **fast** - it's a very productive language
+* It's **easy** to interface to other languages from Python
+* Python has a huge library of existing code to **re-use** like `packet generation <https://www.secdev.org/projects/scapy/>`_ libraries.
+* Python is **interpreted**. Tests can be edited and re-run them without having to recompile the design or exit the simulator GUI.
+* Python is **popular** - far more engineers know Python than SystemVerilog or VHDL
+
+
+*********************
+How does cocotb work?
+*********************
+
+A typical cocotb testbench requires no additional RTL code.
+The Design Under Test (DUT) is instantiated as the toplevel in the simulator without any wrapper code.
+cocotb drives stimulus onto the inputs to the DUT (or further down the hierarchy) and monitors the outputs directly from Python.
+
+
+.. image:: diagrams/svg/cocotb_overview.svg
+
+A test is simply a Python function.
+At any given time either the simulator is advancing time or the Python code is executing.
+The :keyword:`await` keyword is used to indicate when to pass control of execution back to the simulator.
+A test can spawn multiple coroutines, allowing for independent flows of execution.
+
+
+************
+Contributors
+************
+
+.. spelling::
+   McGregor
+   Grimwood
+
+cocotb was developed by `Potential Ventures <https://potential.ventures>`_ with the support of
+`Solarflare Communications Ltd <https://www.solarflare.com/>`_
+and contributions from Gordon McGregor and Finn Grimwood
+(see `contributors <https://github.com/cocotb/cocotb/graphs/contributors>`_ for the full list of contributions).
+
+We also have a list of talks and papers, libraries and examples at our Wiki page
+`Further Resources <https://github.com/cocotb/cocotb/wiki/Further-Resources>`_.
+Feel free to add links to cocotb-related content that we are still missing!
```

Comment:
Please remove the whole contributors section for now, it's outdated. You can replace it with something like:

```rst
cocotb is developed and maintained by an active community under the umbrella of the `Free and Open Source Silicon Foundation <https://www.fossi-foundation.org>`_. 

Our GitHub repository contains a list of all `contributors <https://github.com/cocotb/cocotb/graphs/contributors>`_. 
```

Reply:
I'm not super keen on doing this as part of what is essentially just a move.

I'd rather we revert this for now, and do a separate PR to remove attribution we feel unecessary. Hiding an attribution removal amongst lots of moves feels dishonest (and I plan to squash the commits).

Reply:
Reverted now, we can do it together with the small copyright notices in the HTML footer in conf.py in another PR.